docs/enhancements/YYYYMMDD-kep-template.md

```diff
@@ -18,7 +18,7 @@ see-also:
 replaces:
   - "/docs/enhancements/20181231-replaced-kep.md"
 superseded-by:
-  - "/docs/enhancements/20190104-superceding-kep.md"
+  - "/docs/enhancements/20190104-superseding-kep.md"
```

Comment:
This is a template, the path listed is afaiu intentionally imaginary, but there's no reason for its name not to match the label on the preceding line.